test/Tizen.NUI.Samples/Tizen.NUI.Samples/Samples/FlexibleViewSample.cs

```diff
@@ -229,15 +250,16 @@ public void Activate()
             flexibleView2.FocusLost += FlexibleView_FocusLost;
 
             scrollBar2 = new ScrollBar();
-            scrollBar2.Position2D = new Position2D(2, 194);
-            scrollBar2.Size2D = new Size2D(696, 4);
-            scrollBar2.Style.Track.BackgroundColor = Color.Green;
-            scrollBar2.Style.Thumb.Size = new Size(30, 4);
-            scrollBar2.Style.Thumb.BackgroundColor = Color.Yellow;
-            scrollBar2.Style.Track.ResourceUrl = CommonResource.GetTVResourcePath() + "component/c_progressbar/c_progressbar_white_buffering.png";
+            scrollBar2.Direction = ScrollBar.DirectionType.Horizontal;
+            scrollBar2.Position = new Position(2, 194);
+            scrollBar2.WidthSpecification = 696;
+            scrollBar2.HeightSpecification = 4;
+            scrollBar2.TrackImage.BackgroundColor = Color.Green;
+            scrollBar2.ThumbImage.Size = new Size(30, 4);
+            scrollBar2.ThumbImage.BackgroundColor = Color.Yellow;
+            scrollBar2.TrackImage.ResourceUrl = CommonResource.GetTVResourcePath() + "component/c_progressbar/c_progressbar_white_buffering.png";
             flexibleView2.AttachScrollBar(scrollBar2);
 
-
             FocusManager.Instance.SetCurrentFocusView(flexibleView1);
```

Comment:
Is position of FocusIndicator okay?

When focus is set to View, FocusManager wil add FocusIndicator View to focused view.
So if focused view has LinearLayout, FocusIndicator will be placed at last position of children.

Please check it

Reply:
I have checked with TV. It is normal